app/api/admin_translations.py

```diff
@@ -13,8 +13,9 @@ def download_translations():
     """Admin Translations Downloads"""
     uuid_literal = uuid.uuid4()
     zip_file = "translations{}".format(uuid_literal)
+    zip_file_ext = zip_file+'.zip'
     shutil.make_archive(zip_file, "zip", translations_dir)
```

Comment:
Pass `zip_file_ext` in this function

Reply:
No such file or directory: '/tmp/translations923b7023-08a6-4624-bec7-f8a208698cb5.zip.zip'
This is leading to a wrong filename. The zip downloads are working as of now. Could you please check once again ? @iamareebjamal

Reply:
Done. Sending a PR right now.